doc/manual/manual.tex

```diff
@@ -2268,10 +2268,87 @@ \subsection{Particles}
+\end{enumerate}
+
+\subsection{Docker Container}
+
+\subsubsection{Installing Docker and downloading the \aspect{} image}
+
+Docker is a lightweight virtualization software that allows to ship applications with all their dependencies in a simple way. It is outside of the scope of this manual to explain all possible applications of Docker, and we refer to the introduction (\url{https://www.docker.com/what-docker}) and installation and quickstart guides (\url{https://www.docker.com/products/docker}) on the Docker website for more detailed descriptions of how to set up and use the docker engine. More importantly Docker provides a marketplace for exchanging prepared docker images (called Docker Hub). After setting up the docker engine downloading a precompiled \aspect{} image from Docker Hub is as simple as typing in a terminal:
+
+\begin{verbatim}
+docker pull gassmoeller/aspect
+\end{verbatim}
+
+\subsubsection{Running \aspect{} models}
+Although it is possible to use the downloaded \aspect{} docker image in a number of different ways, we recommend the following workflow:
+
+\begin{enumerate}
+\item Create your \aspect{} input file in a folder of your choice (possibly also containing any input model data that is required by your model).
+\item Run the docker image and mount the prepared directory as a volume into the docker container. The code below assumes you have navigated into the prepared directory. When you have started the container run the aspect model inside the container, and make sure the model output is saved in your prepared directory.
+\begin{verbatim}
+docker run -it -v "$(pwd):/home/dealii/aspect/model" \
+  gassmoeller/aspect:latest bash
+cd model
+../aspect your_input_file.prm
+\end{verbatim}
+
+\item After the model has finished (or during the model run if you want to check intermediate results) the model output will appear in the mounted directory. You can visualize the data outside of the container in your favorite visualization program. Warning: The files are accessible both from within the container and from your host operating system. Try to avoid concurrent write processes.
```

Comment:
Is concurrent writing really an issue and something we need to discuss? I can not think of a case where it matters.